python/ray/data/dataset.py

```diff
@@ -2131,6 +2146,11 @@ def _block_sizes(self) -> List[int]:
         return ray.get(
```

Comment:
should _block_sizes be renamed to _block_num_rows so it's not confused with _block_size_bytes?